### DISCUSSION TITLE
Make Image Magick libraries available on CPPAN

### DISCUSSION THREAD

**Discussion by do-you-dare** [ImageMagick/ImageMagick#6706] [Contribute]
It would be nice to be able to use [cppan](https://cppan.org/) to help setting image magick libraries like `Magick++` and `MagickWand` on C/C++ projects. Is this possible/desirable?

**Comment by mikayla-grace** [comment]
ImageMagick is open source and any of our users are welcome to configure ImageMagick for CPPAN.  The development team is consumed for the foreseeable future and will not have time to pursue your proposal.
